assets/proposals/0264-Separating-the-change-of-Audible-status-and-the-change-of-HMI-Status/index.md

```diff
@@ -0,0 +1,183 @@
+## OnEventChanged
```

Comment:
Can you please revise this file to only include the changes suggested for the `OnEventChanged` section of the HMI Integration Guidelines?  It is not necessary to include the entire file here, only what is planned to be added/removed with the addition of this feature.